.github/workflows/release.yml

```diff
@@ -248,13 +259,14 @@ jobs:
           cd "$(mktemp -d)"
           mv $source/bins-* .
           mkdir dist
+          tag=${GITHUB_REF_NAME//\//}
```

Comment:
This fixes the publish job when running this workflow on a branch for example, right? Could you move this line just a bit higher to where `source` is set as well?